### PR TITLE
Nurse registration app

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: nursereg/serializers.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+from rest_framework import serializers
+from django.contrib.auth.models import User, Group
+from .models import NurseSource, NurseReg
+
+
+class UserSerializer(serializers.HyperlinkedModelSerializer):
```
Comment (imsickofmaps) [inline nursereg/serializers.py]: Drop User and Group please

File: nursereg/urls.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+from django.conf.urls import url, include
+from rest_framework import routers
+from . import views
+
+router = routers.DefaultRouter()
+router.register(r'users', views.UserViewSet)
```
Comment (imsickofmaps) [inline nursereg/urls.py]: Drop as per discussion

**Comment by imsickofmaps** [comment]
minor removals then :+1: when Travis happy

**Comment by imsickofmaps** [comment]
:+1:

**Pull request merged**